target/linux/ramips/image/mt7620.mk

```diff
@@ -616,6 +616,20 @@ define Device/netgear_ex3700
 endef
 TARGET_DEVICES += netgear_ex3700
 
+define Device/netgear_ex6130
+  MTK_SOC := mt7620a
+  NETGEAR_BOARD_ID := U12H319T50_NETGEAR
+  BLOCKSIZE := 4k
+  IMAGE_SIZE := 7744k
+  IMAGES += factory.chk
+  IMAGE/factory.chk := $$(sysupgrade_bin) | check-size $$$$(IMAGE_SIZE) | netgear-chk
+  DEVICE_PACKAGES := kmod-mt76x2
+  DEVICE_VENDOR := NETGEAR
+  DEVICE_MODEL := EX6130
+  SUPPORTED_DEVICES += ex6130
```

Comment:
This is not required when adding a new device. Remove it.

Reply:
I think this section is required because of differen NETGEAR_BOARD_ID

Reply:
I haven't heard about that so far. From what I know, SUPPORTED_DEVICES in only for upgrade from OpenWrt.

Reply:
Will do.
I assumed you're talking about whole block.

Reply:
Have you tried this?

Reply:
@playya 
`SUPPORTED_DEVICES` is not needed for the netgear chk image, it only adds a board to the image-metadata for sysupgrade images. If you add a new board, this happens automagically thru the devices name, as long as the Image definition matches the device-tree compatible (`_` will be substituted internally by `,`).